catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/GazetteerFeatureService.java

```diff
@@ -56,35 +75,82 @@ public void setGeoEntryQueryable(GeoEntryQueryable geoEntryQueryable) {
   }
 
   @Override
-  public Feature getFeatureByName(String name) {
+  public SimpleFeature getFeatureByName(String name) {
+    List<GeoEntry> entries;
     try {
-      List<GeoEntry> entries = this.geoEntryQueryable.query(name, 1);
-      if (!entries.isEmpty()) {
-        GeoResult geoResult = getGeoResultFromGeoEntry(entries.get(0));
-        return getFeatureFromGeoResult(geoResult);
-      }
+      entries = this.geoEntryQueryable.query(name, 1);
     } catch (GeoEntryQueryException e) {
       LOGGER.debug("Error while making feature service request.", e);
+      return null;
+    }
+    if (entries.isEmpty()) {
+      return null;
+    }
+    GeoEntry entry = entries.get(0);
```

Comment:
❓ in what cases would this return more than one entry?  Is it OK to ignore a multi-valued return?

Reply:
`geoEntryQueryable.query` returns a list of GeoEntries that are sorted by name, feature code, etc.  In most cases, a list would be returned for these queries.  `entries.get(0);` would get the most relevant geoEntry for the query.